demo-python/.env-sample

```diff
@@ -10,3 +10,5 @@ COGNITIVE_SERVICES_ENDPOINT=YOUR-COGNITIVE-SERVICES-ENDPOINT
 BLOB_CONNECTION_STRING=YOUR-BLOB-CONNECTION-STRING
 BLOB_CONTAINER_NAME=YOUR-BLOB-CONTAINER-NAME
 FUNCTION_CUSTOM_SKILL_ENDPOINT=YOUR-FUNCTION-CUSTOM-SKILL-ENDPOINT
+INDEX_BLOB_FOLDER=YOUR-INDEXING-DATASET-FOLDER-NAME
+QUERY_BLOB_NAME=YOUR-QUERYING-DATASET-BLOB-NAME
```

Comment:
@farzad528 These two will be present in the container we pass in BLOB_CONTAINER_NAME which is why I believe it would be better to keep it here since we have to configure these values along with container name.